docs/docs/developers/page-editor/index.md

```diff
@@ -9,6 +9,20 @@ content and configuration to the _components_.
 
 ![Page Editor](images/page-editor-overview.png)
 
+## Component actions
+
+When a component is selected the list of available actions is presented in the toolbar.
+Available actions list:
+
+| Title         | Name (technical ID)               | Description                                      |
+|---------------|-----------------------------------|--------------------------------------------------|
+| Properties    | `edit-component-properties`       | Opens dialog to edit a component properties.     |
+| Copy          | `copy`                            | Copy the component.                              |
+| Cut           | `cut`                             | Cut the component.                               |
```

Comment:
Use `Copies`, `Cuts`, `Pastes`, `Changes` as you did with Opens and Removes.

Reply:
done

Reply:
done